published/012-2016-August-24.md

```diff
@@ -0,0 +1,130 @@
+# IPFS Newsletter #11
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened since the IPFS team week in Lisbon in late July. This is the first Newsletter in a while, and there's a ton of exciting content below.
```

Comment:
`addressed by content and identities` doesn't sound that that right, what about

`[IPFS](//ipfs.io/) is a content addressed, hypermedia distribution protocol..`

`aiming to make the web faster` let's take out the `aiming`, it really makes the web faster :)

`In these posts, we highlight some of the development that has happened since the IPFS team week in Lisbon in late July.` I don't feel this is necessary, if we want to highlight it, should be the IPFS Workshop, not team week.